.github/workflows/ci.yml

```diff
@@ -4,13 +4,13 @@ on:
     branches:
       - main
       - dev
-      - temp_dev
   pull_request:
     branches:
       - '*'
 
 env:
   FOUNDRY_PROFILE: ci
+  SEPOLIA_RPC_URL: ${{ secrets.SEPOLIA_RPC }}
```

Comment:
nice
Are you already testing it?

Reply:
There's an issue with the CI running it since there are still conflicts with the base branch, but once they're resolved it should run like magic